docs/topics/mpp/multiplatform-library.md

```diff
@@ -111,9 +111,9 @@ The JS implementation will be very similar to the JVM one.
     
     object JsBase64Encoder : Base64Encoder {
         override fun encode(src: ByteArray): ByteArray {
-            val buffer = js("Buffer").from(src)
-            val string = buffer.toString("base64") as String
-            return ByteArray(string.length) { string[it].toByte() }
+            val string = src.decodeToString()
+            val encodedString = js("btoa(string)") as String
```

Comment:
This can be rewritten without the need for a `js()` invocation:

```kotlin
import kotlinx.browser.window

override fun encode(src: ByteArray): ByteArray {
    val string = window.btoa(src.decodeToString())
    return string.encodeToByteArray()
}
```

Reply:
I've accepted your suggestion, with a minor difference: I've kept the three lines from my original logic. I believe that it would help the beginner Kotlin/JS user, because it clearly shows the three steps the code takes:

1. Convert to the original string.
2. Encode the string using `btoa`.
3. Encode the encoded string as a byte array.

To me it seems that this clearly separates what parts are from the Kotlin stdlib and what from the Kotlin/JS. Let me know what you think! I'd be happy to accept your code as suggested, or just commit it yourself to my PR.

Reply:
Thanks, I like your approach, and your reasoning seems sound. This looks good to me.